epochStart/metachain/systemSCs.go

```diff
@@ -467,7 +467,7 @@ func (s *systemSCProcessor) processSCOutputAccounts(
 			acc.DataTrieTracker().SaveKeyValue(storeUpdate.Offset, storeUpdate.Data)
 		}
 
-		if outAcc.BalanceDelta != nil && outAcc.BalanceDelta.Cmp(zero) > 0 {
+		if outAcc.BalanceDelta != nil && outAcc.BalanceDelta.Cmp(zero) != 0 {
```

Comment:
Is this ok? What happens it BalanceDelta is negative?

Reply:
balancedelta can become negative if you take from a sc and give to some destination. when a sc makes a transfer outAcc balanceDelta is negative.

Reply:
ok. How did it worked up until now? Is this a breaking change?

Reply:
It's not a breaking change.